ui/lib/config-ui/addon/components/messages/page/create-and-edit.js

```diff
@@ -23,7 +23,7 @@ import { isAfter } from 'date-fns';
  */
 
 export default class MessagesList extends Component {
-  @service router;
+  @service('app-router') router;
```

Comment:
this is how you pass the service if the variable you assign it to (i.e. `router`) differs from the service name